environment.yml

```diff
@@ -0,0 +1,166 @@
+name: sd
```

Comment:
I am not sure the changes to the environment are needed. If I am not mistaken, the code should work with the original `environment.yaml` file and if the code is compatible with `diffusers 0.12.1`, then the only change that should be needed is in the `requirements.txt` with updating the diffusers version. We should also add `accelerate`.

Reply:
That's true. I now added accelerate, also just updated transformers version for conveniency.